content/guides/analog-discovery/_index.md

```diff
@@ -0,0 +1,886 @@
+This is an example of using the [Digilent Analog Discovery 3](https://digilent.com/reference/test-and-measurement/analog-discovery-3/start) multi-function test and measurement device to characterize a project on the [Tiny Tapeout 05 ASIC](/runs/tt05/), in particular [#271: Analog Testmacro](https://tinytapeout.com/runs/tt05/tt_um_tt05_analog_test), which is a test analog circuit included on TT05 containing both a ring oscillator and a 3-bit DAC.
+
+
+There is a sample *WaveForms* workspace that is ready to use that can 
+   * talk to the demoboard REPL over serial to setup the correct project;
+   * drive the demoboard inputs to dynamically control the project;
+   * monitor and measure the ring oscillator and DAC outputs; and
+   * graph the results and output CSV files with data.
+
+
+The purpose here is to walk-through the entire process while introducing various features of the AD3 and some of the ways to use it.  If you just want the bullet point list of steps without any details, skip to the end of this page.
+
+
+### WaveForms Installation
+
+To play along, an Analog Discovery 3 is required, as is its front-end application, "Waveforms".  Follow the [AD3 Getting Started Guide](https://digilent.com/reference/test-and-measurement/analog-discovery-3/getting-started#download_and_install_waveforms) to get everything setup and test that it is working.
+
+
+### Running The Full Test Suite
+
+We'll walk through every important step, below, and that is a good way to get to know the Analog Discovery.  However, if you want to just get going and explore things on your own, you may simply wire things up and run the pre-built workspace on the AD3, you may just follow the directions in this section.
+
+## What you need
+
+   1. An Analog Discovery 3 and its *WaveForms* program installed, connected to the computer;
+   2. The *tt05analogdemoFull.dwf3work* workspace opened in *WaveForms*;
+   3. A TT05 demoboard and ASIC, also connected to the computer via USB;
+   4. To have things wired up
+
+The basic wiring is:
+
+```
+AD3     Demoboard Input
+1+      m33 (top header)
+1-      GND (top header, but really anywhere on demoboard)
+2+      m35 (top header)
+2-      GND
+0       in0
+1       in1
+2       in2
+3       in3
+9       in5
+10      in6
+GND     GND
+```
+
+If you also want to run the oscillator frequency vs core voltage characterization, you'll need to
+
+   * depopulate F2 from the demoboard
+   * wire *V+* from the AD3 to the 1V8 pin on the demoboard top header
+
+
+This is optional as you may skip the test at the prompt
+{{< figure src="images/47.png" title="Test run prompt" >}}
+
+or simply get the boring result of no-frequency-variation (since the core voltage won't actually be set by the AD3).
+
+## How to run
+
+
+Assuming you've wired everything up, these are the steps to run the test suite.  Following the entire sequence is only important if you've set things to drive the core supply from the AD3, but it doesn't hurt even if you haven't
```

Comment:
Remove 'assuming you've wired everything up' - as it's already part of the checklist

Reply:
add a : or a full stop at end of sentence